flexmeasures/ui/utils/view_utils.py

```diff
@@ -86,14 +87,27 @@ def render_flexmeasures_template(html_filename: str, **variables):
         options["downloadFileName"] = f"asset-{asset.id}-{asset.name}"
     variables["chart_options"] = json.dumps(options)
 
-    variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+    account: Account | None = (
+        current_user.account if current_user.is_authenticated else None
+    )
+
+    # check if user/consultant has logo_url set
+    if account:
+        variables["menu_logo"] = (
+            account.logo_url
+            or (account.consultancy_account and account.consultancy_account.logo_url)
+            or current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+        )
+    else:
+        variables["menu_logo"] = current_app.config.get("FLEXMEASURES_MENU_LOGO_PATH")
+
     variables["extra_css"] = current_app.config.get("FLEXMEASURES_EXTRA_CSS_PATH")
+    variables["auth_token"] = (
```

Comment:
I get why you implemented this, but can you argue why this is not exposing sensitive information unnecessarily?

Reply:
I understand the concern. Since I'm parsing the `authToken` as part of the server-rendered view, it should only be accessible within the context of that view. This means it's not exposed to the public outside of the view, reducing the risk of unauthorized access. Although, I’m open to any suggestions or alternatives.

Reply:
Thinking about it I'm not sure we need this. Are you sure this doesn't work without sending this to the client?

If the client is logged in, it should already have it. We have other parts in the UI where we talk to our API directly (e.g. loading data into the asset page's graphs, and [the PR which implements pagination](https://github.com/FlexMeasures/flexmeasures/pull/988/files#diff-eca382c76f9f3d77f9d66c17fa35a89ef73c453a404f418875431fba6da8a306R60).

Reply:
Have you tried doing it without (as a logged-in user)?

Reply:
Yeah the API returns an UNAUTHORISED response

Reply:
Yes, but we use it in other places already directly, like [here](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures/ui/templates/base.html#L440) - I believe the browser manages authentication state via a `session` cookie, which the API seems to understand:

![Screenshot from 2024-08-23 22-35-51](https://github.com/user-attachments/assets/1a1241a4-2b40-4eae-b71f-ea8527737d93)

Is your error status 401 or 403? 

I guess before we send around the token, I'd want to test if I have the same issue and we could debug where the auth problem really happens.

Reply:
I'm getting a 401 status

Reply:
Let me look at the reference you linked, maybe I could find the difference between implementations

Reply:
After reviewing the reference, I noticed that no token was passed at all. So, I went ahead and removed that line from my JS script, and it actually works now. Previously, I had removed the token from my HTTP client, which resulted in a 401 error. You must have meant to remove it from the script on the frontend.

Reply:
Good to hear it works as expected now! 

Not sure what you mean with the last sentence. I was concerned with sending the token from server to client, as well as from client to server.

Reply:
You previously asked if i had tested without the token, which I did but on my HTTP client, you must have meant to test in my JS script

Reply:
What is your HTTP client?

Reply:
Insomnia, its like Postman.

Reply:
Ah I heard about it from Victor as well I believe

Reply:
So we can remove that part from this PR right?

Reply:
yeah, already made a commit for it.